### PR TITLE
Feature/reporting

### PR DESCRIPTION
**Pull request by Lance-Simeon**

### CODE REVIEW

File: reporting.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
+# Simeon Cloud Power BI Reporting
+Report on your Simeon Sync data from a single location. Use one of our builtin reports or create your own from the raw data.
+
+## Tenant updates
+When installing Simeon Cloud Power BI Reporting, the installer will make the following changes to the tenant selected to host the Power BI report:
+- Creates an **Azure Service Principal** named Simeon Cloud Power BI Reporting
```
Comment (jeff-simeon) [inline reporting.md:6]: just call "Service Principal" (leave out Azure)

File: reporting.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+- **Generates credentials** for Simeon Cloud Power BI Reporting and saves the credentials in an Azure DevOps Variable Group shared only with your tenant pipelines
```
Comment (jeff-simeon) [inline reporting.md:7]: Generates a client secret....and saves it as a secure variables in an Azure DevOps....

File: reporting.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+- Creates the Simeon Cloud **Power BI Workspace**
+    - Grants admin access to Simeon Cloud Power BI Reporting
```
Comment (jeff-simeon) [inline reporting.md:9]: what does this mean?

File: reporting.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+- Sync Day: The Sync day (used to assist in slicing the data)
+- Sync Time: The time the Sync was run in UTC (used to assist in slicing the data)
+- Configuration: The name of the configuration
+- Configuration Full Name: The full name of the configuration, including the path where the configuration can be found
+- Configuration Description: If the configuration is from the Simeon Baseline this provides a full description about the config
+- Configuration Type: The configuration type used by Simeon to distinguish where a config should be deployed to
+- Configuration Type Description: The translation of the Configuration Type
+- Baseline Name: The name of the baseline the tenant is using. If the tenant does not have a baseline, the value is be **[No Baseline]**
+- Baseline Property Value: The baseline value of the property (if applicable) at the time of the Sync
+- Configuration Reconciliation Type: The section that the configuration falls into on the reconcile screen - **only in tenant**, **available from baseline**, **conflicting with baseline**, **matching baseline**
```
Comment (jeff-simeon) [inline reporting.md:91]: fix capitalization of reconcile screen names

File: reporting.md [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+- Find the pipeline **Backfill Power BI** > **Run pipeline**
+- Select the desired parameters or keep defaults > **Run**
+
+The pipeline parameters are as follows:
+- **Update the Power BI Table schema to the latest schema** keeps the data source schema up to date
+- **Delete all data in the Simeon Power BI dataset** running a data backfill with this unselected will result in duplicate data being uploaded to the data source
+- **Install or reinstall Simeon Power BI report(s)** updates all Simeon reports to the latest version
+- **Backfill only failed upload to Power BI steps** only backfills Syncs that have failed to upload to Power BI, allows you to run the backfill without duplicating data
+- **Number of hours to include in backfill** uploads to Power BI all export/deploy Syncs in the specified number of hours
+- **Number of parallel pipelines to backfill** for most runs should stay as default of 1, shouldn't be greater than 5
```
[the remark this answers is not in the document]
Comment (Lance-Simeon) [reply reporting.md:50]: Because we have done a whole lot of testing on the parallel uploads. Ill remove the update to remove reference to numbers.

**Review by jeff-simeon — DISMISSED** [review]

File: reporting.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (jeff-simeon) [reply reporting.md]: do you mean "have not"? If so, we had better do some - if we run with 1x thread, one of our beta customers will take 100+ hours to do the backfill

**Review by jeff-simeon — DISMISSED** [review]
@david-simeon have you read all of this and approve of the language?

**Review by jeff-simeon — APPROVED** [review]

File: reporting.md [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+- Sync Date: The date of the Sync in UTC time zone
+- Sync Year: The year of the Sync (used to assist in slicing the data)
+- Sync Month: A number representing the month of the Sync (used to assist in slicing the data)
+- Sync Day: The Sync day (used to assist in slicing the data)
+- Sync Time: The time the Sync was run in UTC (used to assist in slicing the data)
+- Configuration: The name of the configuration
+- Configuration Full Name: The full name of the configuration, including the path where the configuration can be found
+- Configuration Description: If the configuration is from the Simeon Baseline this provides a full description about the config
+- Configuration Type: The configuration type used by Simeon to distinguish where a config should be deployed to
+- Configuration Type Description: The translation of the Configuration Type
```
[the remark this answers is not in the document]
Comment (david-simeon) [reply reporting.md:85]: discussing with lance

File: reporting.md [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+### Can I use the dataset to build my own reports?
+Yes, please! If you build a report you think others might like, please let us know and we would be happy to spread the word!
+
+### Why does this dataset not have some of the functions I'm used to in other Power BI datasets?
+We are using a [direct query dataset](https://docs.microsoft.com/en-us/power-bi/connect-data/desktop-use-directquery#benefits-of-using-directquery), which has many benefits including speed and ease of management, but it does come with some limitations. The biggest limitation is the inability to transform the data or to create certain calculated fields. If you find that you need additional data points to build a report, please contact us and we will do what we can to help.
+
+### I don't have a Power BI Pro license, can I still see the workspace?
+Accessing a shared Power BI workspace requires at least a Power BI Pro license. If you are unsure that you want to signup, Power BI does offer a number of trial options. Also, keep in mind, a Power BI Pro license is incuded in the **Office E5 license**.
+
+# Daily Summary Email
```
[the remark this answers is not in the document]
Comment (david-simeon) [reply reporting.md:128]: @Lance-Simeon will be opening a new PR to do so

**Review by david-simeon — DISMISSED** [review]

**Review by jeff-simeon — DISMISSED** [review]

**Review by jeff-simeon — APPROVED** [review]

**Review by rdorta-simeon — APPROVED** [review]

**Review by jeff-simeon — APPROVED** [review]

**Pull request merged**